### PR TITLE
PRIME-1418 Send a message to wallet when credential is revoked

### PR DESCRIPTION
**Pull request by AnaisHebert**

### CODE REVIEW

File: prime-dotnet-webapi/Services/VerifiableCredentialService.cs [hunk 1]
```diff
@@ -19,6 +19,7 @@ public class WebhookTopic
         public const string Connections = "connections";
         public const string IssueCredential = "issue_credential";
         public const string RevocationRegistry = "revocation_registry";
+        public const string BasicMessage = "basicmessages";
```
Comment (mtpultz) [inline prime-dotnet-webapi/Services/VerifiableCredentialService.cs:22]: For consistency with the other this could be snake_cased

Comment (AnaisHebert) [reply prime-dotnet-webapi/Services/VerifiableCredentialService.cs:22]: Yeah I expected it to be snake cased, but that is what they send us

**Review by mtpultz — APPROVED** [review]
image

**Review by hogmann — APPROVED** [review]
🚗 ⛵ 🏠

**Pull request merged**